src/main/java/io/github/railroad/project/Project.java

```diff
@@ -127,18 +131,33 @@ public long getLastOpened() {
         return lastOpened;
     }
 
-    public void open() {
-        System.out.println("Opening project: " + path);
-        this.lastOpened = System.currentTimeMillis();
+    public String getLastOpenedFriendly() {
+
+        Instant instant = Instant.ofEpochMilli(this.getLastOpened());
+        ZonedDateTime zonedDateTime = instant.atZone(ZoneId.systemDefault());
+        ZonedDateTime currentTime = ZonedDateTime.now();
+        long daysDifference = ChronoUnit.DAYS.between(zonedDateTime, currentTime);
+        if (Long.toString(daysDifference).equals("0")) {
+            return "Today";
+        } else {
+            return Long.toString(daysDifference) + " Days ago";
+        }
     }
+    public void setLastOpened(long lastOpened) {
```

Comment:
```suggestion
    }
    
    public void setLastOpened(long lastOpened) {
```
Blank line between methods please